packages/grant-explorer/src/features/api/voting.ts

```diff
@@ -13,6 +22,25 @@ export type PermitSignature = {
   s: string;
 };
 
+export type PermitType = "dai" | "eip2612";
+/** Given a payout token, selects the correct permit type.
+ * - DAI is the old permit type without `value` and with the `allowed` prop
+ * - eip2612 is the standard permit interface, as specified in https://eips.ethereum.org/EIPS/eip-2612
+ *
+ * Old DAI permit type is only implemented on Ethereum and Polygon PoS. Check /docs/DAI.md for more info.
+ * */
+export const selectPermitType = (token: PayoutToken): PermitType => {
+  if (
+    /DAI/i.test(token.name) &&
```

Comment:
Do we already know all the tokens that need this? I'd prefer a list here, "any token that contains the string `dai`" seems a fairly lax check.

Reply:
We have several lists of payout tokens scattered accross the codebase, so a good ticket would be to merge them into one and encode this information there.

Reply:
https://github.com/gitcoinco/grants-stack/issues/2271